src/server/transaction.cc

```diff
@@ -498,8 +498,12 @@ bool Transaction::RunInShard(EngineShard* shard, bool txq_ooo) {
   // If we're the head of tx queue (txq_ooo is false), we remove ourselves upon first invocation
   // and successive hops are run by continuation_trans_ in engine shard.
   // Otherwise we can remove ourselves only when we're concluding (so no more hops will follow).
```

Comment:
This comment does not apply anymore since we don't check for `is_concluding`?

Reply:
I renamed the variable, hope this is more clear now